telearia/docker-compose.yml

```
@@ -0,0 +1,10 @@
+version: "3.7"
+services:
+  telearia:
+    image: ghcr.io/besoeasy/telearia:main@sha256:a24939648693df447cd5e3d40606d7a4bfe495b02bcdfdf619b2275e8f4ebdde
+    restart: unless-stopped
+    ports:
+      - "6799:6799"
+      - "445:445"
```

Port 445 is already used by the Umbrel OS.
```suggestion
      - "6445:445"
```